benefits/core/admin.py

```diff
@@ -17,21 +17,69 @@
 GOOGLE_USER_INFO_URL = "https://www.googleapis.com/oauth2/v3/userinfo"
 STAFF_GROUP_NAME = "Cal-ITP"
 
+logger.debug("Register models with admin site")
+admin.site.register(models.PemData)
 
-for model in [
-    models.AuthProvider,
-    models.EligibilityType,
-    models.PaymentProcessor,
-    models.PemData,
-    models.TransitAgency,
-]:
-    logger.debug(f"Register {model.__name__}")
-    admin.site.register(model)
+
+@admin.register(models.AuthProvider)
+class AuthProviderAdmin(admin.ModelAdmin):
+    def get_exclude(self, request, obj=None):
+        if request.user.groups.contains(Group.objects.get(name=STAFF_GROUP_NAME)):
```

Comment:
This check seems correct, but I wonder if it could be future-proofed just slightly by instead checking if the user is _not_  a superuser, and if not returning these exclude lists.

This will cover the Staff use-case and also the eventual Transit Agency use-case for these fields that should only be visible to superusers.

Reply:
That will work, and I'm ok with changing the check to that. It's easy enough to change later if needed.

Reply:
Yes we can always add more conditions as needed. I think checking the superuser status is a little more explicit for what we are doing here: these are fields that should not be visible to _most_ users.

Reply:
Changed to check if the user is not a superuser: https://github.com/cal-itp/benefits/pull/2224/commits/b8df7b9fe417a7a53a0307e971e1950686b6d6f0